essential-projects/eslint-config
# ♻️ Refactor config to standalone version

## Changes

1. Removes the need for extending from `eslint-config-5minds-typescript/fast`, which in turn extended from `eslint-config-5minds`, which in turn extends from `eslint-config-airbnb-base`
2. Instead, `eslint-config-airbnb-base` is extended directly
3. Declare a full ruleset that reflects the most recent state of the `@essential-projects` ruleset, including all its rule overrides
4. Support ESLint v6
5. Support Typescript v3.7 or higher
6. Since ESLint v6 is a hard dependency of this package, there is no longer any need to have it declared as a dependency in every package that makes use of this config

## Issues

PR: #1

## How to test the changes

- Install this config in a repository that uses ESLint v6 and Typescript v3.7
- See that all linting works correctly
